src/main/resources/io/jenkins/plugins/designlibrary/Links/index.properties

```diff
@@ -1,12 +1,10 @@
-title=Navigational context menu integration
-
 blurb=<p class="jdl-paragraph">Jenkins consists of a large and complex graph of domain objects (<code>ModelObject</code>), where each node in the graph is a web page and edges are hyperlinks. \
   To help users navigate quickly on this graph, Jenkins provides a mechanism to attach context menus to model objects, \
   which can be used to hop multiple edges without going through individual hyperlinks.
 
 define.title=Defining context menu
 
-blurb.define=<p class="jdl-paragraph">To define a context menu on <code>ModelObject</code>, implement <a href="https://javadoc.jenkins.io/byShortName/ModelObjectWithContextMenu" target="_blank"><code>ModelObjectWithContextMenu</code></a>.
+blurb.define=To define a context menu on <code>ModelObject</code>, implement <a href="https://javadoc.jenkins.io/byShortName/ModelObjectWithContextMenu" target="_blank"><code>ModelObjectWithContextMenu</code></a>.
```

Comment:
Rendering of this looks quite broken:
<img width="336" alt="image" src="https://github.com/user-attachments/assets/7188e9bc-e891-4695-a5c5-191a1c571abd" />

Reply:
Title on the page is also: 
<img width="865" alt="image" src="https://github.com/user-attachments/assets/b11c5a97-0e4c-4baa-a568-0ed9c4ea4276" />

Reply:
Try a `mvn clean`, page has been converted from `.groovy` to `.jelly`

Reply:
Ok that fixed it.

Two issues on that page:

1. Can't tell that ModelObjectWithContextMenu is clickable really:
<img width="676" alt="image" src="https://github.com/user-attachments/assets/bd3f18b4-18b9-4d53-8c49-83245e6bc9d9" />
2. syntax highlighting is broken for the java sample (preexisting issue though)

Reply:
Fixed the first one -
<img width="778" alt="image" src="https://github.com/user-attachments/assets/03b92e7e-f1c1-4afd-bf3c-2d3d4b35570c" />

Reply:
fixed the highlighting issue